core/PluginManager.vala

```diff
@@ -1,6 +1,6 @@
 // -*- Mode: vala; indent-tabs-mode: nil; tab-width: 4 -*-
 /*-
- * Copyright (c) 2012-2015 Noise Developers (https://launchpad.net/noise)
+ * Copyright (c) 2012-2018 Noise Developers (https://launchpad.net/noise)
```

Comment:
If we're bumping this better change to `2018 elementary LLC. (https://elementary.io)` since "Noise developers" is not a thing and we're no longer on launchpad